anima/env/mayaEnv/__init__.py

```diff
@@ -294,9 +294,20 @@ def save_as(self, version, run_pre_publishers=True):
                 try:
                     run_publishers(type_name,
                                    publisher_type=PRE_PUBLISHER_TYPE)
-                except PublishError as e:
+                except (PublishError, RuntimeError) as e:
                     # do not forget to clean up the staging area
                     staging.clear()
+
+                    # pop up a message box with the error
+                    pm.confirmDialog(
+                        title='PublishError',
+                        icon='critical',
+                        message='<b>%s</b><br/><br/>%s' % (
+                            'PRE PUBLISH FAILED!!!',
+                            str(''.join([i for i in unicode(e) if ord(i) < 128]))
```

Comment:
you can not use ``unicode`` it is not Python 3.x compatible (for future) and why we are doing this already. You can directly use ``e.message``:

```python
message='<b>%s</b><br/><br/>%s' % (
    'PRE PUBLISH FAILED!!!',
    e.message
),
```

or better use ``e`` directly

```python
message='<b>%s</b><br/><br/>%s' % (
    'PRE PUBLISH FAILED!!!',
    e
),
```
and this is still giving you some errors related to PySide2 you can use ``anima.__string_types__``